pkg/postgres/pgutils/scan.go

```diff
@@ -10,6 +11,19 @@ type Unmarshaler[T any] interface {
 	*T
 }
 
+// ScanRows scans strings.
+//
+// This function closes the rows automatically on return.
+func ScanStrings(rows pgx.Rows) ([]string, error) {
+	return pgx.CollectRows(rows, func(r pgx.CollectableRow) (string, error) {
+		var str string
+		if err := rows.Scan(&str); err != nil {
```

Comment:
nit: I think with the underlying implementation of pgx, it makes no difference, but nevertheless `Scan` should be called on `r` rather than `rows`.